src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Completion/RazorCompletionEndpoint.cs

```diff
@@ -165,6 +165,10 @@ public CompletionRegistrationOptions GetRegistrationOptions()
                 DocumentSelector = RazorDefaults.Selector,
                 ResolveProvider = true,
                 TriggerCharacters = new Container<string>("@", "<", ":"),
+
+                // NOTE: This property is *NOT* processed in O# versions < 0.16
```

Comment:
https://github.com/dotnet/aspnetcore/issues/20320#issuecomment-657901089

Reply:
What does "processed" mean here? Aka, given in this PR that we're removing `:` from the "AllCommitCharacters" in the InitializeHandler Does O# not processing this successfully mean you can't complete completions with : now?

Reply:
https://github.com/dotnet/aspnetcore-tooling/pull/1948#issuecomment-658003481